src/utils.py

```diff
@@ -20,23 +23,43 @@ def __init__(self, data_path):
         download_logger.info(f"Data Handler Class Created - {data_path}")
         self.data_path.mkdir(parents=True, exist_ok=True)
 
-    def downloader(self, download_type):
+    def downloader(self, download_file_name):
         """
-        Downloads download_type in data_path/download_type.
+        Downloads download_file_name in data_path/download_file_name.
         """
-        download_logger.info(f"Downloading {download_type}")
-        fname = f"{download_type}.tar.gz"
+        download_logger.info(f"Downloading {download_file_name}")
+        fname = f"{download_file_name}.tar.gz"
```

Comment:
So glad this `download_file_name`. I was waiting to see how long it takes you to get this fixed :P

Reply:
Well--don't know what to say there :P